packages/familie-skjema/src/felt.ts

```diff
@@ -127,6 +128,7 @@ export function useFelt<Verdi = string>({
     const hentNavBaseSkjemaProps = useCallback(
         (visFeilmelding: boolean): NavBaseSkjemaProps<Verdi> => ({
             feil: visFeilmelding ? feltState.feilmelding : undefined,
+            error: visFeilmelding ? feltState.feilmelding : undefined,
```

Comment:
Kunne man ha fjerna `feil` nå?

Reply:
Siden jeg ikke har helt kontroll på familie-skjema så er jeg usikker på om vi bør fjerne den eller beholde den litt til - for å være "bakoverkompatible". Har jo ikke skrevet om alle komponentene ennå.

Reply:
Nye komponenter vil funke med `Error` mens gamle trenger `Feil`